packages/webapp/src/elections/components/ongoing-election.tsx

```diff
@@ -104,25 +102,25 @@ export const OngoingElection = ({ election }: { election: any }) => {
 };
 
 interface NoFurtherParticipationProps {
-    areRoundsWithNoParticipation?: boolean;
+    ongoingElectionData?: Election;
 }
 
-const NoFurtherParticipationInRoundsMessage = ({
-    areRoundsWithNoParticipation,
+const NoParticipationInFurtherRoundsMessage = ({
+    ongoingElectionData,
 }: NoFurtherParticipationProps) => {
-    if (!areRoundsWithNoParticipation) {
+    if (!ongoingElectionData) return null;
+    if (ongoingElectionData.isMemberStillParticipating) {
         return null;
     }
     return (
-        <Container className="flex items-center space-x-2 pr-8">
+        <Container className="flex items-center space-x-2 pr-8 py-8">
             <BsInfoCircle
                 size={22}
                 className="ml-px text-gray-400 place-self-start mt-1"
             />
             <div className="flex-1">
                 <Text size="sm">
-                    You did not advance a delegate to participate in further
-                    rounds. Please{" "}
+                    You aren't involved in further rounds. Please{" "}
```

Comment:
I'd recommend "aren't" -> "are not" and, in the last sentence, "they are" -> "they will be"? Heck, we could even hide the last sentence when we're in `ElectionStatus.Final`.